Formula/binutils.rb

```diff
@@ -32,6 +38,6 @@ def install
   end
 
   test do
-    assert_match "main", shell_output("#{bin}/gnm #{bin}/gnm")
+    assert_match "__TEXT", shell_output("#{bin}/gstrings #{bin}/gstrings")
```

Comment:
Rather than an using an internal symbol like `__TEXT` (which though unlikely could change) I'd suggest using some text from `strings --help` such as the word `Usage`.

Reply:
That's a great idea!